From fd4eb8b50748b88355d73a8fe4c6b07e6b55ad1a Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Fri, 27 Oct 2023 16:33:13 +0300
Subject: [PATCH 2/2] formatting: use tabs instead of spaces

Signed-off-by: Bohdan Tsybulenko <tsybulenkobohdan@gmail.com>
---
 calculator.cpp | 7 +++++--
 calculator.h   | 6 +++---
 2 files changed, 8 insertions(+), 5 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index 6225df4..268ec06 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -6,10 +6,13 @@ int Calculator::Add (double a, double b)
     return a + b;
 
 	return a + b + 0.5;
- 4ad40a3 (fix truncation error)
+	4ad40a3 (fix truncation error)
+
+	return a + b;
+	cbcb06d (formatting: use tabs instead of spaces)
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.42.0

